CHANGELOG.md

```diff
@@ -40,4 +40,5 @@ See [keep a changelog](https://keepachangelog.com/en/1.0.0/) for information abo
 * Changed sprint report form to GET method.
 * Fixed worklog select path.
 * Optimized worklog select javascript.
+* Fixed issues with create project.
```

Comment:
It would be nice to now what the issue was in the change log! The change log changes er some what short :-)